File: Numeros.py
import threading
import random

class numeros:
    @staticmethod
    def popular_aleatorio(lista, qtd, frase):
        for _ in range(qtd):
            lista.append(random.randint(0, 999))
        print("Feito...." + frase)

    @staticmethod
    def exibir(lista, nome_lista):
        print("Exibindo " + nome_lista + ":")
        for item in lista:
            print(f"{nome_lista}: {item}")

File: C#/Numeros.cs
using System;
using System.Collections.Generic;

public class Numeros
{
    public static void PopularAleatorio(List<int> lista, int qtd, string frase)
    {
        Random gerador = new Random();
        for (int i = 0; i < qtd; i++)
        {
            lista.Add(gerador.Next(1000));
        }
        Console.WriteLine("Feito...." + frase);
    }

    public static void Exibir(List<int> lista, string nomeLista)
    {
        Console.WriteLine("Exibindo " + nomeLista + ":");
        foreach (var item in lista)
        {
            Console.WriteLine(nomeLista + ": " + item);
        }
    }
}